types/node/index.d.ts

```diff
@@ -6367,20 +6362,26 @@ declare module "http2" {
     }
 
     export interface Http2Session extends events.EventEmitter {
-        destroy(): void;
+        readonly alpnProtocol?: string;
+        close(callback?: () => void): void;
+        readonly closed: boolean;
+        destroy(error?: Error, code?: number): void;
         readonly destroyed: boolean;
+        readonly encrypted?: boolean;
+        goaway(code?: number, lastStreamID?: number, opaqueData?: Buffer): void; // TODO
```

Comment:
What is missing to remove the `TODO`?

Also: doc mentions `<Buffer> | <TypedArray> | <DataView>` for `opaqueData`.

Reply:
Yeah -- that was exactly it... it was supposed to be a reminder to myself to change the type. Will fix.

Good catch!